docs/features.md

```
@@ -147,4 +147,15 @@ tests:
 
 These tests can then be run with `cli test`. 
 
-Explore the [Schema](schema.md) for concrete definition of CLI configuration and manifest fields.
\ No newline at end of file
+Explore the [Schema](schema.md) for concrete definition of CLI configuration and manifest fields.
+
+
+## IDE Support
+
+Cliffy's [JSON schema](https://json-schema.org/specification) can be generated locally with `cli init --json-schema`. This will output a `cliffy_schema.json` file which can be referenced in any of your CLI manifest files to enable schema validation and hints. 
```

Markdownlint (0.37.0)

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)